wallet-new/test/MarshallingSpec.hs

```diff
@@ -51,10 +59,21 @@ spec = describe "Marshalling & Unmarshalling" $ do
                 -- currently passphrase should be either empty or of length 32
                 decodingFails @PassPhrase "aabbcc" Proxy
 
+migrateRoundtrip :: (Arbitrary a, Migrate a b, Migrate b a, Eq a, Show a) => proxy a -> proxy b -> Property
+migrateRoundtrip (_ :: proxy a) (_ :: proxy b) = forAll arbitrary $ \(s :: a) -> do
```

Comment:
What about renaming `s` into something a bit more descriptive and `a` and `b` into `from` and `to`? I would be curious to see if this function reads better that way :)